packages/gatsby-plugin-manifest/src/gatsby-ssr.js

```diff
@@ -1,7 +1,12 @@
 import React from "react"
 import { withPrefix } from "gatsby"
+import fs from "fs"
+let cacheId = null
 
-exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
+exports.onRenderBody = (
+  { createContentDigest, setHeadComponents },
```

Comment:
Are we passing `createContentDigest` to SSR APIs?

Reply:
@pieh good catch; nope!

```
[ { setHeadComponents: [Function: setHeadComponents],
    setHtmlAttributes: [Function: setHtmlAttributes],
    setBodyAttributes: [Function: setBodyAttributes],
    setPreBodyComponents: [Function: setPreBodyComponents],
    setPostBodyComponents: [Function: setPostBodyComponents],
    setBodyProps: [Function: setBodyProps] },
  { plugins: [] } ]
```

_should_ we?

Reply:
I'm thinking if we shouldn't provide something like:
```
const { createContentDigest } = require(`gatsby/utils`)
```
for use cases like that (and for tests)

But that said I don't think there is any harm in passing it to ssr APIs - probably in `api-runner-ssr` so we don't need to repeat ourselves for each ssr api?

Reply:
@pieh we could definitely do that. Would just be a little finagling of the transpiled files and an index file in the utils folder (where we can export whatever we _want_ to)

Reply:
I think I prefer the former (i.e. the top level utils file/folder)